addons/web/views/webclient_templates.xml

```diff
@@ -484,7 +484,7 @@
                 <main>
                     <t t-raw="0"/>
                 </main>
-                <footer t-if="not no_footer" id="bottom" data-anchor="true" class="bg-light o_footer">
+                <footer t-if="not no_footer" id="bottom" data-anchor="true" t-attf-class="bg-light o_footer">
```

Comment:
I found out why you removed the bg-light in the first place... you change to t-attf-class here and do not change the xpath removing the bg-light in website. So you removed it here to fix the issue without checking why it was the correct color before :/

Reply:
I'm not sure because I made this before the color system ...